driver/web/adapter.go

```diff
@@ -91,8 +91,10 @@ func (a Adapter) Start() {
 	mainRouter.HandleFunc("/survey-responses", a.wrapFunc(a.clientAPIsHandler.deleteSurveyResponses, a.auth.client.User)).Methods("DELETE")
 	mainRouter.HandleFunc("/survey-alerts", a.wrapFunc(a.clientAPIsHandler.createSurveyAlert, a.auth.client.User)).Methods("POST")
 	mainRouter.HandleFunc("/creator/surveys", a.wrapFunc(a.clientAPIsHandler.getCreatorSurveys, a.auth.client.User)).Methods("GET")
-	mainRouter.HandleFunc("/score", a.wrapFunc(a.clientAPIsHandler.getScore, a.auth.client.User)).Methods("GET")
-	mainRouter.HandleFunc("/scores", a.wrapFunc(a.clientAPIsHandler.getScores, a.auth.client.User)).Methods("GET")
+	mainRouter.HandleFunc("/score", a.wrapFunc(a.clientAPIsHandler.getScoreV1, a.auth.client.User)).Methods("GET")
+	mainRouter.HandleFunc("/scores", a.wrapFunc(a.clientAPIsHandler.getScoresV1, a.auth.client.User)).Methods("GET")
+	mainRouter.HandleFunc("/score/v2", a.wrapFunc(a.clientAPIsHandler.getScoreV2, a.auth.client.User)).Methods("GET")
+	mainRouter.HandleFunc("/scores/v2", a.wrapFunc(a.clientAPIsHandler.getScoresV2, a.auth.client.User)).Methods("GET")
```

Comment:
Hi @akshadpai , typically to handle the `v2` endpoints we would put the `/v2` as the first part of the path (ie. `/v2/score`) to avoid conflicts in the paths for APIs with path variables.